test/test_go.rb

```diff
@@ -923,34 +922,25 @@ def test_history
   def test_execute
     output = '/tmp/fzf-test-execute'
     opts = %[--bind "alt-a:execute(echo /{}/ >> #{output}),alt-b:execute[echo /{}{}/ >> #{output}],C:execute:echo /{}{}{}/ >> #{output}"]
-    wait = ->(exp) { tmux.until { |lines| lines[-2].include?(exp) } }
     writelines(tempname, %w[foo'bar foo"bar foo$bar])
-    tmux.send_keys "cat #{tempname} | #{fzf(opts)}; sync", :Enter
-    wait['3/3']
+    tmux.send_keys "cat #{tempname} | #{fzf(opts)}", :Enter
+    tmux.until { |lines| lines[-2] == '  3/3' }
```

Comment:
`execute` action switches to the alternate screen, so I was concerned that the keys we send may not register to fzf. But I could be wrong. Do you think it's fine without waits?

Reply:
Hmm, in theory I'm unsure whether, in this scenario, the keys might not register to fzf. However I've run the tests in an indefinite loop several times, for 50 minutes at a time, and not seen this happen, so I'm confident that, at least in practice, they're not needed.

Reply:
Got it, let's remove them. We can always restore them when we see errors again.